Doc/reference/datamodel.rst

```diff
@@ -1243,9 +1243,10 @@ Methods on code objects
 
    The iterator returns :class:`tuple`\s containing the ``(start_line, end_line,
    start_column, end_column)``. The *i-th* tuple corresponds to the
-   position of the source code that compiled to the *i-th* instruction.
-   Column information is 0-indexed utf-8 byte offsets on the given source
-   line.
+   position of the source code that compiled to the *i-th* code unit. A
+   code unit is either a bytecode instruction or a cache entry of an
```

Comment:
To be pedantic, cache entries can span several code units, as can the instruction if the `EXTENDED_ARG` prefix is included.

Maybe just use "code unit" here, and explain the instruction format elsewhere (in another PR)